Source/Core/Common/Config/Config.cpp

```diff
@@ -69,6 +70,7 @@ void RemoveLayer(LayerType layer)
 
 ConfigChangedCallbackID AddConfigChangedCallback(ConfigChangedCallback func)
 {
+  std::unique_lock lock(s_callbacks_lock);
```

Comment:
These can be a simple `std::lock_guard` instead of `std::unique_lock`, not that it makes a huge difference.